### PR TITLE
Replaced go:generate with go:embed

### LINKED ISSUE #737: Embed static assets in Skychat
**Feature description**

Senyoret created a PR for improving the Skychat UI but we need to embed the static assets into the binary similar to the way we embed the hypervisor assets into the visor.

### PR DESCRIPTION
**Pull request by ersonp**
Did you run `make format && make check`?

Fixes #737 

 Changes:	
- replaced //go:generate with //go:embed in skychat

How to test this PR:
1. Set up [Interactive test environments](https://github.com/SkycoinPro/skywire-services/blob/master/docs/InteractiveEnvironments.md).
2. `git checkout` this PR branch.
3. Follow the instructions to Test with local changes from [Interactive test environments](https://github.com/SkycoinPro/skywire-services/blob/master/docs/InteractiveEnvironments.md).
4. Run `make integration-run-generic`
5. Go to `http://localhost:8001/` and `http://localhost:8002/` and check if its working as intended.

### CODE REVIEW

File: cmd/apps/skychat/chat.go [hunk 1]
```diff
@@ -56,7 +59,7 @@ func main() {
 	conns = make(map[cipher.PubKey]net.Conn)
 	go listenLoop()
 
-	http.Handle("/", http.FileServer(FS(false)))
+	http.Handle("/", http.FileServer(getFileSystem(false)))
```
Comment (jdknives) [inline cmd/apps/skychat/chat.go]: What are we using the live mode for? Not even sure if we need it. @Senyoret1 did you ever use that?

Comment (ersonp) [reply cmd/apps/skychat/chat.go]: The previous version had the option so I kept it for the new one.

**Comment by i-hate-nicknames** [comment]
Great idea. I would also add a comment where does `//go:embed static` points to in the project.
Does `make format` and `make check` upgrade dependencies? I tried to run those on develop branch and they didn't. Not sure how upgrading packages is related to embedding, can you elaborate please?

**Comment by ersonp** [comment]
> Does `make format` and `make check` upgrade dependencies? I tried to run those on develop branch and they didn't. Not sure how upgrading packages is related to embedding, can you elaborate please?

The packages were upgraded because I was having issues with `github.com/incu6us/goimports-reviser` in `make install-linters` which installs an older version of `goimports-reviser`. That has been fixed and merged in developed now. I will revert the upgrades.

**Pull request merged**